### PR TITLE
[EDU-5503] feature: add guide - How to query the top URLs impacted by bots

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-5503] 

### Changes

- Add  How to query the top URLs impacted by bots guide EN/PT.
- Add the link to the Guides page.
- Add the label and link to the GraphQL menu.

### Additional links

n/a.

[EDU-5503]: https://aziontech.atlassian.net/browse/EDU-5503?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gabriel-alves-azion — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/graphql/bot-manager-breakdown-data.mdx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+:::note
+Para recuperar esses dados, você deve estar inscrito no Azion Bot Manager e usar a [API GraphQL](/pt-br/documentacao/devtools/graphql-api/visao-geral/). Entre em contato com a [equipe de vendas](https://www.azion.com/pt-br/contate-vendas/) para mais detalhes sobre a assinatura.
+:::
+
+---
+
+## Querying data
+
+To query the top 5 URLs impacted by bot activity, proceed as follows:
```
Comment (evandroazion) [inline src/content/docs/pt-br/pages/guias/graphql/bot-manager-breakdown-data.mdx:24]: Aqui não deveria estar em PT-BR?

Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/graphql/bot-manager-breakdown-data.mdx:24]:
```suggestion
Para consultar as 5 principais URLs impactadas pela atividade de bots, proceda da seguinte forma:
```

File: src/content/docs/pt-br/pages/guias/graphql/bot-manager-breakdown-data.mdx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+1. Access the GraphiQL playground in this link: `https://manager.azion.com/metrics/graphql`.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/graphql/bot-manager-breakdown-data.mdx:26]:
```suggestion
1. Acesse o GraphiQL playground nesse link: `https://manager.azion.com/metrics/graphql`.
```

File: src/content/docs/pt-br/pages/guias/graphql/bot-manager-breakdown-data.mdx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    - You must be logged in to your Azion account. Otherwise, you'll receive an error message.
+2. Send a query following this format:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/graphql/bot-manager-breakdown-data.mdx:28]:
```suggestion
    - Você deve estar logado na sua conta Azion. Caso contrário, você receberá uma mensagem de erro.
2. Envie uma query seguindo este formato:
```

**Review by guiafonso-ol — APPROVED** [review]

**Review by brunomrpx — APPROVED** [review]

**Review by robsongajunior — APPROVED** [review]

**Pull request merged**